app/models/changelog.php

```diff
@@ -122,3 +122,20 @@
 
     return $text;
 };
+
+$latest_undocumented_release = function() use ($releases, $github_link, $release_title) {
+  $text = '';
+  $version_number = str_replace("\n", '', str_replace('v', '', file_get_contents(CACHE_PATH . 'tag.txt')));
```

Comment:
No need to replace `\n`,  read the file with `FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES` flags. 
I would also just strip the first character from the tag.